docs/guide/tutorial-8-the-supervisor.md

```diff
@@ -1,10 +1,6 @@
 ## Tutorial 8: the Supervisor (30 Minutes)
 
-A [Supervisor](../reference/supervisor.md) is a special type of [Robot](../reference/robot.md) which has additional powers.
-In fact, any [Robot](../reference/robot.md) can be turned into a supervisor simply by setting its field named `supervisor` to TRUE.
-A [Supervisor](../reference/supervisor.md) can modify the environment by adding or removing nodes to the scene, it can change their properties by modifying the values of parameters in a programmatic way, allowing for instance to move or setup a robot a certain way, it can start the creation of movies, animations and, last but not least, thanks to its unlimited access it can be used to acquire measurements about the state of the simulation as well as track its evolution.
-
-This tutorial will explore how to achieve some of these tasks using a [Supervisor](../reference/supervisor.md).
+This tutorial will explore how to achieve some tasks using a [Supervisor](../reference/supervisor.md). A Supervisor oversees the world to set or get information about it. To fully understand what is it for, read the description of [Supervisor](../reference/supervisor.md) in the [Webots Reference Manual](https://www.cyberbotics.com/doc/reference/introduction).
```

Comment:
```suggestion
A [Supervisor](../reference/supervisor.md) oversees a world and can set or get information about it.
This tutorial will teach you:
- How to move objects using the supervisor.
- How to delete a node and spawn another in its place.
- How to track the evolution of an object in the scene.
- How to change the properties of an object, namely its color.

These are only some of the powers of the [Supervisor](../reference/supervisor.md), to fully understand what it is capable of, you should refer to its documentation. 
```

Reply:
I would write something more like this, so that people know what the tutorial is about and if it interests them.
I haven't checked the formatting, so the test documentation test might not pass.